package/yast2-firewall.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-firewall
-Version:        3.1.4
+Version:        3.1.5
```

Comment:
below in spec file you have to increase version for yast2 dependecy to ensure you have yast2 with firewalld support